src/admin/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -24,7 +24,7 @@ const Sidebar: FunctionComponent<SidebarProps> = ({ headerLink, navItems }) => (
 						<NavLink
 							activeClassName="o-sidebar__nav-item--active"
 							className="o-sidebar__nav-item"
-							to={navItem.location}
+							to={navItem.location as string}
```

Comment:
Is er een bepaalde voorkeur in volgende:
- `${navItem.location}`
- navItem.location as string
- String(navItem.location)

Reply:
* ${navItem.location} => gebruik ik enkel om te concatineren, nooit om te casten naar een string.
* navItem.location as string gebruik ik als ik zeker weet dat die variabele een string is maar als de typescript compiler het niet door heeft. Dit verandert het type niet in javascript.
* Dit gebruik ik om een number om te zetten naar een string. Omdat het zo duidelijk toont wat je doet. (${}) doet dit veel minder.